configure.ac

```diff
@@ -994,6 +994,9 @@ AC_REPLACE_FUNCS(mkdtemp)
 AC_CHECK_DECLS(strrstr)
 AC_REPLACE_FUNCS(strrstr)
 
+AC_CHECK_DECLS(stpncpy)
+AC_REPLACE_FUNCS(stpncpy)
+
```

Comment:
I would expect that something would have to be changed in *libcompat/Makefile.am* to add the new source file, but somehow this is enough, right?

Reply:
I would so too, but I think there's some magic going on with the filenames in that directory. I'm pretty sure it worked at the time I introduced it.